helix-rest/src/test/java/org/apache/helix/rest/server/TestHelixRestServer.java

```diff
@@ -59,7 +59,7 @@ public void testInvalidHelixRestServerInitialization() {
     try {
       List<HelixRestNamespace> invalidManifest3 = new ArrayList<>();
       invalidManifest3.add(
-          new HelixRestNamespace("DuplicatedName", HelixRestNamespace.HelixMetadataStoreType.ZOOKEEPER, ZK_ADDR, true));
+          new HelixRestNamespace("DuplicatedName", HelixRestNamespace.HelixMetadataStoreType.ZOOKEEPER, ZK_ADDR, false));
```

Comment:
If a namespace is set to default, both COMMON_SERVLET and DEFAULT_SERVLET will be created. COMMON_SERVLET being created means "/namespaces/default" will be created.
Since this commit is to delete "/namespaces/default", we don't set a default namespace for the test here and it is not necessary. We are just validating duplicate names.